src/components/v2/Layout/Header/index.tsx

```diff
@@ -18,10 +19,12 @@ export const Header = ({ location }: HeaderProps) => {
     [location.pathname],
   );
   const styles = useStyles();
+  const { t } = useTranslation();
+
   return (
     <AppBar position="relative" css={styles.appBar}>
       <Toolbar css={styles.toolbar}>
-        <h3>{title}</h3>
+        {title && <h3>{t(title)}</h3>}
```

Comment:
In this case, the title would be passed as a string directly so no need to wrap it with `t()`